js/load-scene.js

```diff
@@ -97,15 +97,19 @@ AFRAME.registerComponent('loadscene', {
       browserReposition();
     }
 
+    function onStartClick() {
+      introAudioEl.pause();
```

Comment:
in fact, can we destroy `introAudioEl` if we will never use it again in this page lifecycle?

Reply:
Probably? But because it's in `a-assets`, I don't want to pull the rug from under it.